### ISSUE TITLE
some problem

### ISSUE THREAD

**Issue by luckydog11** [JierunChen/FasterNet#10]
I follow your idea to train my model using dataset about face， but results are comparable to resnet（params are comparable）， what do you think may be the problem？Thank you .

**Comment by JierunChen** [comment]
Hi, if you are replacing the regular Conv in ResNet with PConv, don't forget to remove the activation layer as it helps the ResNet but hurts the expressiveness of the variant with PConv. 

If you are comparing FasterNet with ResNet, you may adjust the model hyperparameters, e.g., the partial ratio which reflects the redundancy to exploit, and some training hyperparameters, e.g., the lr, weight decay, and drop_path_rate. Different models may vary in favored training recipes.

**Comment by luckydog11** [comment]
> Hi, if you are replacing the regular Conv in ResNet with PConv, don't forget to remove the activation layer as it helps the ResNet but hurts the expressiveness of the variant with PConv.
> 
> If you are comparing FasterNet with ResNet, you may adjust the model hyperparameters, e.g., the partial ratio which reflects the redundancy to exploit, and some training hyperparameters, e.g., the lr, weight decay, and drop_path_rate. Different models may vary in favored training recipes.

I just run your model, but using different dataset, like face data.  Results are comparable to ResNet（params are comparable）, I don't mean I replace the regular Conv in ResNet with PConv, I just mean I compare the results of different models, the results of PConv seems to be no advantage， I don't know why.

**Comment by JierunChen** [comment]
> > Hi, if you are replacing the regular Conv in ResNet with PConv, don't forget to remove the activation layer as it helps the ResNet but hurts the expressiveness of the variant with PConv.
> > If you are comparing FasterNet with ResNet, you may adjust the model hyperparameters, e.g., the partial ratio which reflects the redundancy to exploit, and some training hyperparameters, e.g., the lr, weight decay, and drop_path_rate. Different models may vary in favored training recipes.
> 
> I just run your model, but using different dataset, like face data. Results are comparable to ResNet（params are comparable）, I don't mean I replace the regular Conv in ResNet with PConv, I just mean I compare the results of different models, the results of PConv seems to be no advantage， I don't know why.

Then as I said previously, you may adjust the model hyperparameters, e.g., the partial ratio which reflects the redundancy to exploit, and/or some training hyperparameters, e.g., the lr, weight decay, and drop_path_rate. Different models may vary in favored training recipes.

**Comment by xiaboAAA** [comment]
> 我按照你的想法使用人脸数据集训练我的模型，但结果与resnet相当（参数相当），你认为可能是什么问题？谢谢。

Hello, what dataset are you running?
